src/shared/components/Uploader/Uploader.tsx

```diff
@@ -194,6 +213,11 @@ export const Uploader = ({
           image={image}
         />
       )}
+      <RemoveImagePopup
+        open={!!isRemovePopupOpen}
```

Comment:
why double exclamation ('!!') but not just isRemovePopupOpen?

Reply:
fixed